upgrade/1.2/plan_and_coordinate_network_upgrade.md

```diff
@@ -106,6 +106,14 @@ referring to user activity.
    and allows for better easing into BICAN in CSM 1.2. For more details about this transition plan, see
    [Minimize UAN Downtime](../../operations/network/management_network/bican_enable.md#minimize-uan-downtime).
 
+## Manually removing UAN switch ports from the CMN VLAN7 (Placeholder)
+
+After the upgrade to UAN 2.4, the UAN switch ports should be removed from CMN VLAN7 to prevent user traffic from being able to reach endpoints on the CMN.  In the CSM 1.2 and UAN 2.4 upgrade, this removal is not done automatically.  A future release or hotfix to CSM will introduce this automation.
```

Comment:
```suggestion
After the upgrade to UAN 2.4, the UAN switch ports should be removed from CMN VLAN7 to prevent user traffic from being able to reach endpoints on the CMN.
In the CSM 1.2 and UAN 2.4 upgrade, this removal is not done automatically. A future release or hotfix to CSM will introduce this automation.
```